app/components/works/add_files_component.html.erb

```diff
@@ -10,9 +10,9 @@
   <div class="form-check" data-complex-radio-target="selection">
     <%= form.radio_button :upload_type, "browser", class: "form-check-input", "data-file-uploads-target": "browserRadioButton",
           data: {action: "complex-radio#disableUnselectedInputs file-uploads#updatePanelVisibility deposit-button#updateDepositButtonStatus"} %>
-    <%= form.label :upload_type_browser, "Upload fewer than 250 files (total less than 10 GB) to be displayed as a flat list of files.", class: "form-check-label fw-semibold" %>
+    <%= form.label :upload_type_browser, "Upload one or more files (less than 10GB) to be displayed as a flat list of files.", class: "form-check-label fw-semibold" %>
     <div id="uploaded-files-panel" data-file-uploads-target="fileUploads">
-      <div data-controller="dropzone" data-dropzone-max-file-size="10000" data-dropzone-max-files="250" data-dropzone-previews-container=".dropzone-files-previews">
+      <div data-controller="dropzone" data-dropzone-max-file-size="10000" data-dropzone-previews-container=".dropzone-files-previews">
```

Comment:
similar question here for [this change](https://github.com/sul-dlss/happy-heron/pull/3199/files#diff-8891d5301b70a836983b39503029abbd3839576f77e9280c10d413a92a9b48aaR15)